woocommerce/wc-admin
# Load Analytics API behind feature check

Depends on https://github.com/woocommerce/woocommerce-admin/pull/7168

Enable the Analytics API's only if the feature is turned on and add a new REST endpoint to return available features.

### Detailed test instructions:

1. Confirm the new endpoint works by making a request to `/wp-json/wc-admin/features`
2. Delete the analytics option `wp option delete woocommerce_analytics_enabled` and see the endpoint results reflect a default to "yes" for analytics.
3. Set the option value to "no" and see the endpoint reflect that.
4. Change a config or directly edit `incluces/feature-config.php` and confirm the endpoint reflects that as well.
5. With analytics option toggle turned off, confirm a request to an analytics endpoint results in 404, ie `/wp-json/wc-analytics/leaderboards`.
